examples/org-viewer-react/src/OrgApps.tsx

```diff
@@ -0,0 +1,31 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import { useApps } from '@aragon/connect-react'
+import Group from './Group'
+import Table from './Table'
+import TextButton from './TextButton'
+
+type Props = {
+  onOpenApp: (address: string) => void
+}
+
+export default function OrgApps({ onOpenApp }: Props) {
+  const [apps, { loading }] = useApps()
+
+  return (
+    <Group name="Apps" loading={loading}>
+      <Table
+        headers={['Name', 'Version', 'Address']}
+        rows={[...apps]
+          .sort((a, b) => (a.name || '').localeCompare(b.name || ''))
+          .map(app => [
+            app.name || 'unknown',
+            app.version || '?',
+            <TextButton onClick={() => onOpenApp(app.address)}>
+              {app.address.slice(0, 6)}
```

Comment:
I saw we're doing this in a few places, but should we use the `0xabcd....1234` format instead?

Know it's a demo, but still better to espouse good practices :).